src/main/java/seedu/address/model/card/Card.java

```diff
@@ -75,6 +76,10 @@ public Schedule getSchedule() {
         return schedule;
     }
 
+    public List<String> getOptions() {
+        return null;
```

Comment:
?

Reply:
no choice, need the parent class `Card` to have that function so that the `MCQ card` can override it